src/app/core/nav-menu/components/app-nav/app-nav.component.html

```diff
@@ -0,0 +1,24 @@
+<div>
+  <cs-menu-header (buttonClicked)="close.emit(true)"></cs-menu-header>
```

Comment:
Why emit `true`?

Reply:
Because EventEmitter has a boolean type, I just need to send anything.